src/components/sidebar-database/sidebar-database.less

```diff
@@ -81,7 +81,8 @@
 
     &-is-active,
     &-is-active:hover {
-      background-color: @compass-sidebar-active-background-color;
+      background: @compass-sidebar-base-background-color;
+      @compass-sidebar-base-background-color: lighten(hsl(201, 11%, 41%), 20%); // item-is-active
```

Comment:
@fredtruman for some reason I'm not able to see the active state, but this is the color that it's supposed to adopt.

Reply:
@durran think i might need that piece of code to view the instance header after all